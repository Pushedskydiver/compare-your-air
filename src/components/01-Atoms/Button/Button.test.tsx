import React from 'react';
import { cleanup, render } from '@testing-library/react';;
import ButtonStyles from './Button.styles';

afterEach(() => {
  cleanup();
});

describe('<ButtonStyles />', () => {
  it('should match snapshot', () => {
      const { container } = render(
        <ButtonStyles>This is a button</ButtonStyles>
      );
      
      expect(container).toMatchSnapshot();
  })
})