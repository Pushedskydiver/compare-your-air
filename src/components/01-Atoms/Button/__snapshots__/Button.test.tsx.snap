// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonStyles /> should match snapshot 1`] = `
<div>
  <button
    class="sc-AxjAm bHHMnX"
  >
    This is a button
  </button>
</div>
`;
